pom.xml

```diff
@@ -335,17 +342,11 @@
                 <artifactId>maven-jar-plugin</artifactId>
                 <version>3.2.2</version>
             </plugin>
-            <plugin>
-                <artifactId>maven-deploy-plugin</artifactId>
-                <version>3.1.1</version>
-                <configuration>
-                    <!--altDeploymentRepository>internal.repo::default::file://${project.build.directory}/mvn-repo</altDeploymentRepository-->
-                </configuration>
-            </plugin>
+
```

Comment:
Please put the old version, I need it for the CI and it already works fine

Reply:
You meant this section?